### DISCUSSION TITLE
Issue Discussion

### DISCUSSION THREAD

**Discussion by abskrj** [ZeStream/zestream-server#18] [General]
Hey team, created 2 new issues #16 and #17, let me know if you would like to take

**Comment by abskrj** [comment]
let me know, which one you would like to take up @uzaxirr

**Reply by uzaxirr** [reply]
I'll be happy to tackle on #16 

But for that we need to get #8  resolved i suppose?

**Reply by abskrj** [reply]
I will pick #8, if not fixed before tomorrow noon

**Reply by uzaxirr** [reply]
I would be happy to pick up #8 followed by #16 as they are co-related

**Comment by abskrj** [comment]
Hey team, quick question, is it better to use Swagger for documentation or use something like [Docsify](https://docsify.js.org/#/deploy)?

**Reply by abskrj** [reply]
cc: @uzaxirr

**Reply by uzaxirr** [reply]
Haven’t used Docsify ever before, I'm good with Swagger.
